adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -202,7 +141,8 @@ private void initialize(Context appContext, String authority, ITokenCacheStore t
         if (mTokenCacheStore != null) {
             mTokenCacheAccessor = new TokenCacheAccessor(mTokenCacheStore, mAuthority);
         }
-        mJWSBuilder = new JWSBuilder();
+
+        mAcquireTokenRequest = new AcquireTokenRequest(appContext, validateAuthority, mTokenCacheAccessor);
```

Comment:
pass in the token cache store, and acquireTokenRequest will be responsible for initializing the cache accessor.